### PR TITLE
#135 update to java 11

### PR DESCRIPTION
**Pull request by occupant23**

### CODE REVIEW

File: src/main/java/com/xceptance/neodymium/module/StatementBuilder.java [hunk 1]
```diff
@@ -35,6 +36,22 @@
...
+            throw new RuntimeException(e);
+        }
+        catch (NoSuchMethodException e)
+        {
+            throw new RuntimeException(e);
+        }
+        catch (SecurityException e)
+        {
+            throw new RuntimeException(e);
+        }
```
Comment (oomelianchuk) [inline src/main/java/com/xceptance/neodymium/module/StatementBuilder.java:54]: Why don't you catch all exceptions in one multi-catch clause?

Comment (occupant23) [reply src/main/java/com/xceptance/neodymium/module/StatementBuilder.java]: done

**Review by oomelianchuk — APPROVED** [review]
IMO everything is fine

**Comment by occupant23** [comment]
@h-arlt I adjusted the code. Please review.

**Review by h-arlt — APPROVED** [review]
Looks good to me.

**Pull request merged**